src/common/wincmn.cpp

```diff
@@ -634,26 +634,16 @@ void wxWindowBase::FitInside()
 }
 
 // On Mac, scrollbars are explicitly children.
-#if defined( __WXMAC__ ) && !defined(__WXUNIVERSAL__)
```

Comment:
I'm not sure why is this not Mac-only any more?

Reply:
There are a few reasons:

1.  There is no (relevant) performance hit.
2. IsClientArea is a public method for all ports. I have not checked whether this is Mac-only or if this method will be Mac-only till the end of wxWidgets. And I do not have to worry: the modification will work in any case (as long as IsClientAreaChild exists).
3. IMHO it is difficult to maintain exceptions in a source code that are related to specific ports only. This can be seen from the fact that in this case a bug has been introduced. Perhaps, somebody was not familiar with the Mac port or forgot that grow boxes are also children. Or the grow box became a child after the implementation of scrollbars as children and the person making the change did not know that somewhere else in the code there is at least one assumption that scrollbars are children but not real ones (actually, the same bug occurred twice(!) in the same source code file, in the second case no port distinction has been made (cast to scrollbars only) and I detected it by pure coincidence).

PS: If scrollbars or grow windows are children or not is another topic, I think. And you agree, I believe, 
fixing this may take much longer. This is an appropriate fix till then.